libs/components/lists/src/lib/modules/list-summary/list-summary-item.component.ts

```diff
@@ -0,0 +1,54 @@
+import { Component, TemplateRef, computed, input } from '@angular/core';
+import { SkyNumericModule, SkyNumericOptions } from '@skyux/core';
+import { SkyKeyInfoModule } from '@skyux/indicators';
+
+/**
+ * Displays a single summary statistic with a label and formatted value.
```

Comment:
```suggestion
 * Displays a list summary item that pairs a label with a formatted value.
```
I don't think we need "single" since "a" indicates that this is singular. And I'd use "list summary item" given that we agreed to use that language in the Design tab guidance and elsewhere. We could add "to highlight important information" to the end if you think it adds value, but I'm not sure we need it.

Reply:
I used a modified version of the corresponding doc comment for the key info component here. I think this comment is intended to summarize what a `sky-list-summary-item` is, not restate that it is a list summary item.

Reply:
Which JSDocs comment from key info are you referring to? I checked those docs but don't see the corresponding comment in the docs at https://developer.blackbaud.com/skyux/components/key-info?docs-active-tab=development.

We can definitely stick with this current wording if you prefer, but "a single summary statistic" just felt awkward/stilted to me. I wonder if it would align with both of our preferences if we rearranged this something more like:

"Displays a summary detail about the list that pairs a label with a formatted value."

But if you still prefer the current version, I'd just stick with that and resolve this comment. Thanks!